src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java

```diff
@@ -110,9 +116,10 @@ public void deleteInstanceTest() throws FogbowManagerException, ClientProtocolEx
         this.openStackAttachmentPlugin.deleteInstance(instanceId, this.localToken);
         
     }
-
+    
+    // FIXME: this method does nothing
```

Comment:
I guess the idea was to actually forward the exception thrown in  this.client.doDeleteRequest(endpoint, localToken); I suggest you to delete the method and to add a TODO mark explaining that

Reply:
The test was properly rewritten